simplq/src/components/ErrorHandler.jsx

```diff
@@ -1,27 +1,34 @@
-/* eslint-disable */ // todo enable it back
-
 import React from 'react';
-import PageNotFound from './pages/PageNotFound';
 import { setErrorPopupMessage } from 'store/appSlice';
 import { store } from 'store';
+import * as Sentry from '@sentry/react';
+import PageNotFound from './pages/PageNotFound';
 
+// eslint-disable-next-line import/prefer-default-export
 export class ErrorBoundary extends React.Component {
   constructor(props) {
     super(props);
     this.state = { hasError: false };
   }
 
-  static getDerivedStateFromError(error) {
+  static getDerivedStateFromError() {
     // Update state so the next render will show the fallback UI.
-    console.log('In getDerivedStateFromError');
+    // eslint-disable-next-line no-console
+    console.log(
+      'Something caused a crash during rendering, falling back to 404 url, please try again...'
+    );
     return { hasError: true };
   }
 
   componentDidCatch(error, errorInfo) {
     // You can also log the error to an error reporting service
     // logErrorToMyService(error, errorInfo);
-    console.log('In componentDidCatch');
     store.dispatch(setErrorPopupMessage('An error occured. Please try again'));
+    // log error to sentry for alerting
+    Sentry.withScope((scope) => {
+      scope.setExtras(errorInfo);
+      Sentry.captureException(error);
+    });
   }
```

Comment:
Sentry doesn't seem to pick up the exception. It might be because our quota ended.